exercises/practice/nucleotide-count/.docs/instructions.append.md

```diff
@@ -0,0 +1,8 @@
+# Instructions append
+
+The output should be a String having 4 numbers separated by spaces. Each number represents a count for A, C, G and T in this order.
+
+For example:
+```javascript
+NucleotideCounts.parse('GATTACA') -> '3 1 1 2'
```

Comment:
There was another PR that changed this signature. I updated this for you and also tweaked the sentence above a little.